### ISSUE TITLE
UI Injectors don't change hair color

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#11406]
(WEB REPORT BY: dtscurvy REMOTE: 172.93.109.202:7777)

> Revision 9debda602789f89c86f5106da2808e629554f274
> 
> After being cloned, if you attempt to put your corpse into a genetics scanner to get the UI to restore your original hair color, the color still doesn't change.
> 
> I expected the UI injector to restore the hair and facial hair color of what I looked like before I was cloned.
> 
> The UI doesn't change shit, even if you get somebody to put you in the scanner to change the UI blocks, supposedly because as opposed to those blocks being shifted into another number when cloned, the properties of those blocks are changed into another hair/facial hair color.
> 
> Get cloned, put the corpse into a genetics scanner, get the UI injector

**Comment by Exxion** [comment]
#5054
TL;DR the injectors DO change the hair color, but don't change it to the correct color for some reason, and the color they change it to is the same color your hair changes to when you're cloned
